helm_deploy/django-app/values.yaml

```diff
@@ -7,27 +7,30 @@ replicaCount: 1
 image:
   repository: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/cloud-platform-demo-app
   tag: latest
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
 
 service:
   type: ClusterIP
-  port: 80
+  port: 8000
 
 ingress:
   enabled: true
   annotations: {}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
   path: /
-  hosts:
-    - helm-demo.apps.cloud-platforms-test.k8s.integration.dsd.io
   tls: []
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
 
+deploy: 
+  hosts: helm-demo.apps.cloud-platforms-test.k8s.integration.dsd.io
```

Comment:
Having a default value here could likely be a pitfall for users, because if they don't override it they'll likely end up colliding with another install of the app, which isn't an obvious thing to debug - so I'd say remove the default and mark this is a required value with Helm.

Reply:
My plan was to be explicit in the documentation, showing how command line arguments overwrite values in values.yaml. e.g. `helm install --set deploy.host=django-jason.apps.cloud-platforms-test.k8s.integration.dsd.io`

Reply:
I can see that, but given that this is a value that users _have_ to set I'd say it's better to mark this as a required value, and use something like the Deployment replicas field as an example of a value with a default that users might want to override.

Reply:
Fair point. Changed to an empty string in values.yaml. Under b8760fb.

Reply:
👍 this is the ideal place to try out the `required` helper function too, as an empty string on its own won’t actually prompt the user to provide a value - can you give that a go? https://github.com/kubernetes/helm/blob/master/docs/charts_tips_and_tricks.md